fw-child/resources/js/child-functions.js

```diff
@@ -530,4 +517,19 @@ const $ = jQuery;
 
     $('body').removeClass('spinner-on');
   });
+
```

Comment:
Why did you move this code out of the Vendor section where it was?

Reply:
I restored the code to the place where it was.